README.md

```diff
@@ -1 +1,158 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Hello-Microverse</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+**[Hello-Microverse]** This project is a Hello Microverse repository. No complex coding is required for this exercise. The goal is to master all of the tools and best practices learned in previous steps. They will be used in all Microverse projects and most likely in future jobs as well, so it is important to know!.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>front end</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+- added index.html 
+- added style.css
+
+- **[Display Hello-Microverse]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need the following tools:
+- Browser
+- IDE(e.g.: Vscode,...)
+- Terminal
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/Young-Gaius/Hello-Microverse
+
+### Install
+
+- Install this project with:
+
+```sh
+  cd mv_hello-microverse
+
+  npm install or npm i
+
```

Comment:
- [ ]  Please add closing back ticks here (```). As suggested by previous reviewer, your readme is breaking because it doesn't have closing back ticks at the appropriate places.